vuepress/docs/next/tutorials/consume/solr.md

```diff
@@ -0,0 +1,168 @@
+
+To employ Solr, the index and configuration files called the core, has to be created. This tutorial decribes the instructions for both the general and multitenant use cases. For a mulitenant application, a core has to be generated for the primary and each of the secondary tenants. For secondary tenants, see the [Multitenacy on Entando] tutorial.
+
+## Prerequisites
+* [A working instance of Entando](../../docs/getting-started/README.md). with the default Tomcat server image
+* Verify dependencies with the [Entando CLI](../../docs/getting-started/entando-cli.md): `ent check-env develop`
+* [Helm](https://helm.sh/docs/intro/install/) to handle Solr installation
+* For multitenant applications, the [Content Delivery Server] is required 
+
+## Create the Descriptor
+1. Install Solr:  
+```
+helm repo add apache-solr https://solr.apache.org/charts
+helm repo update
+```  
+For more information, see the [Solr Getting Started page](https://solr.apache.org/guide/7_3/solr-tutorial.html).
+
+2. Install the Solr and Zookeeper CRDs:
+```
+kubectl create -f https://solr.apache.org/operator/downloads/crds/v0.5.0/all-with-dependencies.yaml
+```
+3. Install the Solr operator and Zookeeper Operator:
+```
+helm install solr-operator apache-solr/solr-operator --version 0.5.0
+```
+4. Create a descriptor file `deployment-solrCloud.yaml` following the sample below. Use the domain name of your Entando application.
+
+For example:
+``` yaml
+apiVersion: solr.apache.org/v1beta1
+kind: SolrCloud
+metadata:
+  name: solr
+spec:
+  solrAddressability:
+    external:
+      domainName: YOUR-DOMAIN
+      method: Ingress
+      useExternalAddress: true
+  customSolrKubeOptions:
+    podOptions:
+      resources:
+        limits:
+          memory: 2Gi
+          cpu: 1000m
+        requests:
+          cpu: 350m
+          memory: 2Gi
+  dataStorage:
+    persistent:
+      pvcTemplate:
+        spec:
+          resources:
+            requests:
+              storage: 10Gi
+      reclaimPolicy: Delete
+  replicas: 3
+  solrImage:
+    repository: entando/entando-solr
+    tag: "8"
+  solrJavaMem: -Xms2048M -Xmx2048M
+  updateStrategy:
+    method: StatefulSet
+  zookeeperRef:
+    provided:
+      chroot: /explore
+      image:
+        pullPolicy: IfNotPresent
+        repository: pravega/zookeeper
+        tag: 0.2.13
+      persistence:
+        reclaimPolicy: Delete
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+      replicas: 3
+      zookeeperPodPolicy:
+        resources:
+          limits:
+            memory: 500Mi
+          requests:
+            cpu: 250m
+            memory: 500Mi
+
+```
+
+3. Apply the Solr application YAML:
+```
+kubectl apply -f deployment-solrCloud.yaml -n entando
```

Comment:
@avdev4j In most of the tutorials we use -n entando because it's the default in a quickstart env and that means the command can be copied unmodified. We expect admins will know to replace the namespace with their own.